litellm/model_prices_and_context_window_backup.json

```diff
@@ -2430,6 +2430,38 @@
         "supports_reasoning": true,
         "supports_tool_choice": true
     },
+    "azure/o3-deep-research": {
+        "max_tokens": 100000,
+        "max_input_tokens": 200000,
+        "max_output_tokens": 100000,
+        "input_cost_per_token": 1e-05,
+        "output_cost_per_token": 4e-05,
```

Comment:
lgtm - verified here: https://azure.microsoft.com/en-us/blog/introducing-deep-research-in-azure-ai-foundry-agent-service/